tests/integration/easy-slot-picker/component-test.js

```diff
@@ -0,0 +1,84 @@
+import {module, test} from 'qunit';
+import {setupRenderingTest} from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+import { generateAppointmentSlots } from 'ember-appointment-slots-pickers/test-support/helpers/generate-appointment-slots';
+import Ember from 'ember';
+
+import {render, settled, findAll} from '@ember/test-helpers';
+
+const emberAssertCp = Ember.assert; // eslint-disable-line
+
+module('Integration | Component | easy-slot-picker', function (hooks) {
+  setupRenderingTest(hooks);
+
+  hooks.beforeEach(function () {
+    //silence Ember.assert for other components presence, check didInsertElement hook of the component
+    Ember.assert = () => {}; // eslint-disable-line
```

Comment:
can we add a todo to get rid of this!

Reply:
Yes I have a to-do already to revisit the default loaders bit and those assertions (it's still copy pasted code so don't intend to change too much things but there are some weird parts there so will do it).